packages/template-ui/src/views/Search.vue

```diff
@@ -4,7 +4,11 @@
     :style="{ backgroundImage: backgroundImageURL }"
   >
     <ChannelNavBar />
-    <SearchBar v-model="query" @clear-input="onClearInput" />
+    <SearchBar
+      v-model="query"
+      :debounce="2000"
```

Comment:
This PR shows that this delay constant is hardcoded in two components. Let's take this opportunity to move it to the constants.js module, so next time it can be edited in a single place.